rpc/gnmi/mixed-schema.md

```diff
@@ -99,6 +96,35 @@ constrained to the specified origin. Particularly:
   the specified `origin`. To delete contents from multiple origins, a client
   MUST specify multiple paths within the `delete` of the `SetRequest`.
 
+### Special considerations for the use of Origin CLI
+
+The following rules apply to a setRequest containing `origin` CLI.  TODO:
+Special considerations for mixing native schema are currently undefined.
+
+* If a `SetRequest` contains `origin` `cli` and `origin` `openconfig`, the CLI
+  MUST be given precedence over the OpenConfig origin.  This precedence is
+  independent of the order in which origin CLI appears.  This overrides the
+  gNMI specification requirement of ordered path based precedence within a
+  `SetRequest`.  The goal of this precedence is to create deterministic
+  behavior for resolving overlapping configuration.  For setRequests without
+  `origin` CLI, order based precedence still applies.
```

Comment:
This last sentence seems to somewhat contradict what we converged on for #474/#740. I thought that it wasn't so much about origin = CLI as origin = *any native schema* (CLI or modelled)?

Reply:
+1 to @jsterne comment above - what `cli` really means is "native schema" + ascii_val encoding.  I've always been of the opinion it should be named as such to allow for multiple encodings of a native schema.  `cli` is an improper origin name especially in comparison to `openconfig` which can encode data per the appropriate TypedVal

But the wording above implies that if origin=openconfig + native modeled data as say JSON_IETF (which would fall into another origin) loops right back around to the same problem

Reply:
I think we want to explicitly maintain support for origin `cli`.    We could elaborate on the definition of what origin `cli` means in a separate pull request to update  [gNMI 2.2.2](https://github.com/openconfig/reference/blob/master/rpc/gnmi/gnmi-specification.md#222-paths) where the data types and origins are defined.

Reply:
To answer @jsterne question, the intention is this section only applies to origin `cli`.   The definition of origin `cli` is admittedly not very strictly defined.

Reply:
It feels like this "OpenConfig replace" + "Native update" should be defined to work (in a similar way) whether Native CLI or Native YANG modelled is used. At least for the basic concepts like having the native take precedence,

Reply:
To be more concrete: I feel pretty strongly that this sentence should not be part of the spec: "For setRequests without `origin` CLI, order based precedence still applies."

Reply:
I can remove that sentence and leave that topic open/unclarified for now.  We should not bring in vendor yang mixing issues in this PR.